### PR TITLE
feat: deprecate isOptimisticMirroredByMe, use isMirroredByMe instead

### PR DESCRIPTION
**Pull request by krzysu**

### CODE REVIEW

File: examples/web-wagmi/src/publications/UseCreateMirror.tsx [hunk 1]
```diff
@@ -33,14 +33,13 @@ function MirrorInner({ publisher }: MirrorInnerProps) {
...
 
-  const isMirroredByMe = publication.isOptimisticMirroredByMe || publication.mirrors.length > 0;
-
   return (
     <div>
       <PublicationCard publication={publication} />
 
       <div>Total Mirrors: {publication.stats.totalAmountOfMirrors}</div>
-      <div>Is Mirrored by Me: {isMirroredByMe ? 'true' : 'false'}</div>
+      <div>My mirrors: {publication.myMirrors.length}</div>
```
Comment (cesarenaldi) [inline examples/web-wagmi/src/publications/UseCreateMirror.tsx:41]: ❤️

File: packages/api-bindings/src/apollo/cache/__tests__/ApolloCache.spec.ts [hunk 2]
```diff
@@ -518,5 +525,103 @@ describe(`Given an instance of the ${ApolloCache.name}`, () => {
...
+              }),
+            }),
+          ]);
+
+          const { writePublication, readPublication } = setupApolloCache();
+          writePublication(publication);
+          const read = readPublication(publication);
+
+          expect(read.stats.totalAmountOfCollects).toEqual(2);
+        });
```
Comment (cesarenaldi) [inline packages/api-bindings/src/apollo/cache/__tests__/ApolloCache.spec.ts:623]: 🙏 thx

File: packages/api-bindings/src/apollo/cache/createContentPublicationTypePolicy.ts [hunk 3]
```diff
@@ -100,10 +104,73 @@ const hasCollectedByMe = (existing: boolean, { readField }: FieldFunctionOptions
...
+  });
+
+  const collectPendingTx = getAllPendingTransactions().find((transaction) => {
+    return isCollectTransactionForThisPublication(transaction);
+  });
+
+  return {
+    ...existing,
+    totalAmountOfMirrors: existing.totalAmountOfMirrors + mirrorPendingTxs.length,
+    totalAmountOfCollects: existing.totalAmountOfCollects + (collectPendingTx ? 1 : 0),
```
Comment (cesarenaldi) [inline packages/api-bindings/src/apollo/cache/createContentPublicationTypePolicy.ts:160]: Can you please clarify why this is not using the count?

File: packages/react/src/transactions/adapters/responders/__tests__/CollectPublicationResponder.spec.ts [hunk 4]
```diff
@@ -80,14 +96,15 @@ describe(`Given the ${CollectPublicationResponder.name}`, () => {
       stats: knownInitialPublicationStats,
     }),
   ])('and a $__typename', (publication) => {
-    describe(`when "${CollectPublicationResponder.prototype.commit.name}" method is invoked`, () => {
+    describe.only(`when "${CollectPublicationResponder.prototype.commit.name}" method is invoked`, () => {
```
[the remark this answers is not in the document]
Comment (krzysu) [reply packages/react/src/transactions/adapters/responders/__tests__/CollectPublicationResponder.spec.ts:99]: whoops, my bad

File: packages/api-bindings/src/apollo/cache/createContentPublicationTypePolicy.ts (hunk 3 above)
Comment (cesarenaldi) [reply packages/api-bindings/src/apollo/cache/createContentPublicationTypePolicy.ts]: not, they behave the same, more than one collect in progress can exist for the same publication.

File: examples/web-wagmi/src/publications/UseCreateMirror.tsx (hunk 1 above)
Comment (krzysu) [reply examples/web-wagmi/src/publications/UseCreateMirror.tsx:41]: I'd just leave them as `mirrors` then, less maintenance, and just explain that they are observer mirrors in the docs, unfortunately, we can't really auto-document fragment fields

Comment (cesarenaldi) [reply examples/web-wagmi/src/publications/UseCreateMirror.tsx:41]: ok

**Review by cesarenaldi — APPROVED** [review]

File: examples/web-wagmi/src/publications/UseCreateMirror.tsx (hunk 1 above)
Comment (krzysu) [reply examples/web-wagmi/src/publications/UseCreateMirror.tsx]: removed `myMirrors` for now

**Pull request merged**